test/python/SMO_script.ps1

```diff
@@ -0,0 +1,53 @@
+$argss = $args[0]
+$paramsArray =$argss.Split("?")
+
+$Assem = (“Microsoft.SqlServer.Management.Sdk.Sfc”, 
+“Microsoft.SqlServer.Smo”, 
+“Microsoft.SqlServer.ConnectionInfo”,
+“Microsoft.SqlServer.SqlEnum”);
+
+Add-Type -AssemblyName $Assem
+
+$SmoServer = New-Object ('Microsoft.SqlServer.Management.Smo.Server') -argumentlist $paramsArray[0]
+$SmoServer.ConnectionContext.LoginSecure = $false
+$SmoServer.ConnectionContext.set_Login($paramsArray[3])
+$SmoServer.ConnectionContext.set_Password($paramsArray[4])
+$db = $SmoServer.Databases[$paramsArray[2]]   
+
+$schm = "sys"
+$dtb = "sysdatabases"
+
+$Objects = $db.Tables                                                                
+$Objects += $db.Views         
+$Objects += $db.StoredProcedures
+$Objects += $db.UserDefinedFunctions
+$SubObjects += $db.Tables.Indexes
+$SubObjects += $db.Tables.Triggers
+
+foreach ($CurrentObject in $Objects) 
+{     
+    if ($CurrentObject.schema -ne $schm -and $CurrentObject.schema -ne $dtb -and $CurrentObject.schema -ne $null -and -not $CurrentObject.IsSystemObject )
```

Comment:
I think now we can update the condition to 
```
 if (-not $CurrentObject.IsSystemObject)
```
Since the issue with system views being considered as User defined view has been fixed now.